interfaces/vrep/DQ_VrepRobot.m

```diff
@@ -4,13 +4,11 @@
 % Usage:
 %   Inherit from this class and implement the abstract methods.
 %
-%   DQ_VrepRobot Methods:
-%       send_q_to_vrep - Sends the joint configurations to VREP
-%       get_q_from_vrep - Obtains the joint configurations from VREP
-%       kinematics - Obtains the DQ_Kinematics implementation of this
-%       robot
+%   DQ_VrepRobot Methods (Abstract):
+%       set_configuration_space_positions - Sends the joint configurations to VREP
+%       get_configuration_space_positions - Obtains the joint configurations from VREP
```

Comment:
Please change the methods' names according to my comments on the `DQ_SerialVrepRobot.m` file.

Reply:
Names changed in [0cf51a6](https://github.com/dqrobotics/matlab/pull/108/commits/0cf51a6b35c6a49f577d5aa3494ed10e7aa136f0).